scalyr_agent/agent_status.py

```diff
@@ -751,8 +751,8 @@ def __report_copying_manager(output, manager_status, agent_log_file_path, read_t
                 )
 
         if (
-            worker_pool.has_good_health_checks is not None
-            and not worker_pool.has_good_health_checks
+            worker_pool.all_health_checks_good is not None
+            and not worker_pool.all_health_checks_good
```

Comment:
`None` is not truthy, so can't you just write this as `not worker_pool.all_health_checks_good`?  (i.e., eliminate the first predicate that checks for `is not None`)

Reply:
The main idea here is to not print anything if there is a None. 
Some fields like the health check and response status are not determined immediately from the start, so we just do not show them while they are unknown.  
In the code above there is a case when not all responses are successful and we want to show unsuccessful ones.

But I have to agree that I could make things smaller by "worker_pool.all_health_checks_good is False"